discovery-provider/src/tasks/entity_manager/user.py

```diff
@@ -123,9 +124,31 @@ def create_user(params: ManageEntityParameters):
         is_current=False,
     )
 
-    if params.metadata == "v2":
+    user_metadata = None
+    try:
+        # for single tx signup
+        user_metadata = json.loads(params.metadata)["data"]
+        validate_user_metadata(
+            params.session,
+            user_record,
+            user_metadata,
+        )
+
+        user_record = update_user_metadata(
+            params.session,
+            params.redis,
+            user_record,
+            user_metadata,
+            params.web3,
+            params.challenge_bus,
+        )
+    except Exception:
+        # fallback to multi tx signup
+        pass
+
+    if params.metadata_cid == "v2":
```

Comment:
should be `params.metadata` now bc of this change https://github.com/AudiusProject/audius-protocol/commit/f17be75f8c95c8838f02b2288cc3465e72559dfa